docs/modules/ROOT/nav.adoc

```diff
@@ -23,6 +24,11 @@
 ** xref:jet-engine-configuration.adoc[Configuring the Jet Engine]
 ** xref:jet-job-configuration.adoc[Running Data Pipelines]
 * xref:custom-config.adoc[Custom Configuration of Hazelcast cluster]
+* Management Center
+** xref:management-center-clusters.adoc[Hazelcast Clusters]
+** xref:management-center-external-access.adoc[External Access]
+** xref:management-center-persistance.adoc[Persistence]
```

Comment:
It's not a big issue but I need to mention it :) 
Could you fix the typo for `persistance`? There are multiple places. Sample yaml file name as well.